### PR TITLE
913178: Documentation of hierarchical binding topic in hotfix

### PR DESCRIPTION
**Pull request by Vinitha-Balasubramanian**
Included the hierarchical binding topic in hotfix branch

### CODE REVIEW

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/hierarchy-grid.md [hunk 1]
```diff
@@ -0,0 +1,804 @@
...
+
+By default, the parent and child grid relation is maintained using the [QueryString](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.Grids.Grid.html#Syncfusion_EJ2_Grids_Grid_QueryString) property, which requires the same field name for both grids. However, to achieve the parent and child relation with different fields, you need to modify the mapping value in the [Load](https://help.syncfusion.com/cr/aspnetmvc-js2/Syncfusion.EJ2.Grids.Grid.html#Syncfusion_EJ2_Grids_Grid_Load) event of child grid. 
+
+In the following example, the `Load` event is utilized to customize the mapping value for the child grid. By accessing the `parentDetails` property and its **parentKeyFieldValue**, you can set the desired mapping value. The `parentRowData` property contains the data of the parent row, and by using the **EmployeeID** field name, you can extract the corresponding value from the parent row data.
```
Comment (MaithiliyK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.MVC/hierarchy-grid.md:42]: is the casing is correct

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/hierarchy-grid.md [hunk 2]
```diff
@@ -0,0 +1,827 @@
...
+{% endhighlight %}
+{% highlight c# tabtitle="Childgrid.cs" %}
+{% include code-snippet/grid/hierarchy-grid/hierarchy-grid/html.cs %}
+{% endhighlight %}
+{% endtabs %}
+
+![Hierarchy grid](../images/hierarchy-grid/hierarchy-grid.png)
+
+> * Grid supports n level of child grids.
+> * Hierarchical binding is not supported when [DetailTemplate](https://ej2.syncfusion.com/aspnetcore/documentation/grid/row/detail-template) is enabled.
```
Comment (MaithiliyK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/hierarchy-grid.md:37]: APi casing issue

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/hierarchy-grid.md [hunk 3]
```diff
@@ -0,0 +1,827 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+![Render aggregates in child grid](../images/hierarchy-grid/aggregates-child-grid.png)
+
+## Expand all by external button
+
+The Hierarchy Grid in the Syncfusion ASP.NET Core Grid component allows you to expand all child grid rows using an external button. This feature provides you with a convenient overview of all the hierarchical data within the grid, eliminating the need to manually expand each row individually.
+
+By default, Grid renders all child grid rows in collapsed state. To expand all child grid rows in the Grid using an external button, you can utilize the `expandAll` method provided by the DetailRow module. Similarly, to collapse all grid rows, you can use the `collapseAll` method.
```
Comment (MaithiliyK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/hierarchy-grid.md:198]: API missing

**Review by MaithiliyK — CHANGES_REQUESTED** [review]
address my review comments

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/hierarchy-grid.md (hunk 1 above)
Comment (Vinitha-Balasubramanian) [reply ej2-asp-core-mvc/grid/EJ2_ASP.MVC/hierarchy-grid.md:46]: Yes, it's an internal property of child grid. Please refer to the added image 
image

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/hierarchy-grid.md (hunk 2 above)
Comment (Vinitha-Balasubramanian) [reply ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/hierarchy-grid.md]: It's not an API reference, referring ug documentation link of detail template.

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/hierarchy-grid.md (hunk 3 above)
Comment (Vinitha-Balasubramanian) [reply ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/hierarchy-grid.md:200]: There are no API links for grid methods for both core and mvc.

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Pull request merged**